test/e2e/framework/auth/helpers.go

```diff
@@ -77,8 +77,13 @@ func WaitForNamedAuthorizationUpdate(c v1beta1authorization.SubjectAccessReviews
 	return err
 }
 
-// BindClusterRole binds the cluster role at the cluster scope
-func BindClusterRole(c v1beta1rbac.ClusterRoleBindingsGetter, clusterRole, ns string, subjects ...rbacv1beta1.Subject) {
+// BindClusterRole binds the cluster role at the cluster scope. If RBAC is not enabled, nil
+// is returned with no action.
+func BindClusterRole(c v1beta1rbac.ClusterRoleBindingsGetter, clusterRole, ns string, subjects ...rbacv1beta1.Subject) error {
+	if !isRBACEnabled {
+		return nil
```

Comment:
If we only want to hide the error because "RBAC may simply be disabled" lets just test it as such.